normalizer.py

```diff
@@ -34,6 +34,7 @@ def craft(self, docs: DocumentArray, **kwargs) -> DocumentArray:
         filtered_docs = DocumentArray(
             list(filter(lambda d: 'image/' in d.mime_type, docs))
         )
+        self.read(filtered_docs)
         for doc in filtered_docs:
```

Comment:
```suggestion
        for doc in filtered_docs:
            self._convert_image_to_blob(doc)
```

Reply:
Addressed, please see [cbdeeb4](https://github.com/jina-ai/executor-image-normalizer/pull/11/commits/cbdeeb4baba0007618176dc489dc6d59780ed9a7). Thank you.